src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -22,25 +22,24 @@ private sealed class Logger
         {
             private TextWriter _logger;
             private LogLevel _requestedLogLevel;
-            private string _projectPath;
+            private string _fileName;
 
             public Logger(TextWriter logger, LogLevel requestedLogLevel, string projectPath)
             {
                 _logger = logger;
                 _requestedLogLevel = requestedLogLevel;
-                _projectPath = projectPath;
+                _fileName = Path.GetFileNameWithoutExtension(projectPath);
             }
 
             private void Log(LogLevel level, string message, params object[] values)
             {
                 if (level <= _requestedLogLevel)
                 {
-                    _logger?.WriteLine("FastUpToDate: " + string.Format(message, values) + " (" + Path.GetFileNameWithoutExtension(_projectPath) + ")");
+                    _logger?.WriteLine($"FastUpToDate: {string.Format(message, values)} ({Path.GetFileNameWithoutExtension(_fileName)})");
```

Comment:
You dont need `Path.GetFileNameWithoutExtension(_fileName)` instead just `_fileName` because `Path.GetFileNameWithoutExtension` is already applied on `_fileName`